### ISSUE TITLE
Missing PSVita swizzling

### ISSUE THREAD

**Issue by SAOMDVN** [bartlomiejduda/ImageHeat#2]
Hi, it seems like PSVIta swizzling is supported in ReverseBox, but isn't available in this GUI.

**Comment by bartlomiejduda** [comment]
Hi. Do you have any swizzled textures for testing?
Function in ReverseBox was not tested properly. I need some samples to refactor the code first.

**Comment by SAOMDVN** [comment]
[tex_all.zip](https://github.com/user-attachments/files/17953373/tex_all.zip)

Hi, this is a texture file from a PSVita game.

**Comment by SAOMDVN** [comment]
Thank you a lot! Do you know what format is the correct one for the file I sent above? I tried W=1024, H=512, Start Offset=16, Format=RGBA8888, Little Endian and PSVita swizzle but don't quite get the right result

**Comment by bartlomiejduda** [comment]
No, sorry, I couldn't preview your sample file either. I was doing my tests on GXT files from game "SENRAN KAGURA SHINOVI VERSUS" on PS Vita. Then I have created my own GXT file using existing tools.
